language/Portuguese_Brazil/strings.po

```diff
@@ -416,30 +416,30 @@ msgstr ""
 
 #: src/frontend/mame/ui/info.cpp:176
 msgid "Completely unemulated features: "
-msgstr ""
+msgstr "Características completamente não emuladas"
 
 #: src/frontend/mame/ui/info.cpp:182 src/frontend/mame/ui/info.cpp:198
 #, c-format
 msgid "%s"
-msgstr ""
+msgstr "%s"
 
 #: src/frontend/mame/ui/info.cpp:182 src/frontend/mame/ui/info.cpp:198
 #: src/frontend/mame/ui/info.cpp:243
 #, c-format
 msgid ", %s"
-msgstr ""
+msgstr "%s"
```

Comment:
This is for appending to a list.  If you don't put space/punctuation, you'll end up with text like:
```
Características completamente não emuladasdiscoREDEWAN
```